### ISSUE TITLE
JSHint 'server' is not defined

### ISSUE THREAD

**Issue by chrisdpeters** [samselikoff/ember-cli-mirage#401]
I upgraded to Ember CLI 1.13.12 (and then 1.13.13), and I get this JSHint error for all of my acceptance tests now:

```
1. acceptance/page/new-test.js should pass jshint.
acceptance/page/new-test.js: line 20, col 3, 'server' is not defined.
```

Any ideas on what may be causing this? It's difficult to sift through all of these failed tests to find the "real" failures that I need to address.

Using version 0.1.11 of ember-cli-mirage if that makes any difference.

**Comment by jherdman** [comment]
It sounds to me like your `tests/.jshintrc` "predef" isn't complete. Here's a snippet of mine:

```json
{
  "predef": [
    "server",
    "document",
    "window",
    "location",
```

See my entry for "server"? You need to add it to your copy of this file.

**Comment by chrisdpeters** [comment]
@jherdman Perfect. I just noticed that in my "Upgrade to Ember CLI 1.13.12" commit, both `/.jshintrc` and `tests/.jshintrc` were modified to remove `server` from the list. I probably thought it was safe to remove that when I ran `ember init` after upgrading CLI. (Now I know better.)

In an earlier attempt, I had added it back to the root `.jshintrc` file but not the one in `tests` and thought I was going crazy. When you do that, the PhantomJS runner works fine, but not the Chrome one. Thanks for helping me ultimately figure it out.
